tests/baselines/reference/user/chrome-devtools-frontend.log

```diff
@@ -3090,11 +3097,14 @@ node_modules/chrome-devtools-frontend/front_end/bindings/BreakpointManager.js(60
         Types of parameters 'debuggerModel' and 'model' are incompatible.
           Type 'T' is not assignable to type 'DebuggerModel'.
 node_modules/chrome-devtools-frontend/front_end/bindings/BreakpointManager.js(97,56): error TS2345: Argument of type 'string' is not assignable to parameter of type 'K'.
+  'string' is assignable to the constraint of type 'K', but 'K' could be instantiated with a different subtype of constraint '{}'.
```

Comment:
better error message. I wonder why the constraint is `{}` and not `unknown`. Maybe we don't have strictNullChecks on?